src/platform_impl/android/mod.rs

```diff
@@ -796,8 +796,9 @@ impl CoreWindow for Window {
 
     fn pre_present_notify(&self) {}
 
-    fn inner_position(&self) -> Result<PhysicalPosition<i32>, RequestError> {
-        Err(NotSupportedError::new("inner_position is not supported").into())
+    fn surface_position(&self) -> PhysicalPosition<i32> {
+        // FIXME: Complete this implementation
```

Comment:
I'm not really a fan of having `FIXME` like that in the code, when the proper issue could be opened.

Reply:
I've opened https://github.com/rust-windowing/winit/issues/3910